<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="game"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>27</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Peli</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionRestart"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>K채ynnist채</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>K채ynnist채 uudestaan</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Lopeta</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>Tietoja Qt:sta</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
